protocol/0082-ETHD-ethereum-data-source.md

```diff
@@ -110,84 +110,84 @@ Select {
 
 ### External Oracles - Creation
 
-1. Create ethereum oracles based on calling a read method of a smart contract (Phase 2 - oracle based on listening for events) (<a name="0081-ETHD-001" href="#0081-ETHD-001">0081-ETHD-001</a>)
-2. All current governance rules that apply to propose / submit / vote on a proposal should be applicable for the ethereum oracle data source creation / amendment (<a name="0081-ETHD-002" href="#0081-ETHD-002">0081-ETHD-002</a>)
-3. Create more than spam.protection.max.proposals oracle data source proposals in an epoch - proposal rejected with error message (<a name="0081-ETHD-003" href="#0081-ETHD-003">0081-ETHD-003</a>)
-4. Create ethereum oracles based on calling a read method of a smart contract by supplying incorrect ABI (Phase 2 - oracle based on listening for events) (<a name="0081-ETHD-004" href="#0081-ETHD-004">0081-ETHD-004</a>)
+1. Create ethereum oracles based on calling a read method of a smart contract (Phase 2 - oracle based on listening for events) (<a name="0082-ETHD-001" href="#0082-ETHD-001">0082-ETHD-001</a>)
+2. All current governance rules that apply to propose / submit / vote on a proposal should be applicable for the ethereum oracle data source creation / amendment (<a name="0082-ETHD-002" href="#0082-ETHD-002">0082-ETHD-002</a>)
```

Comment:
Is there a specific data source creation proposal or is this talking about proposals such as market creation/amendment that introduce a new Eth oracle?

Reply:
Yes, the governance path that creates/updates a market is the intended way to handle this kind of data source, in the same way as the other ones.